testonly/hammer/hammer.go

```diff
@@ -124,14 +125,14 @@ type MapBias struct {
 	InvalidChance map[MapEntrypointName]int
```

Comment:
should `safePRNG` be part of `MapBias`?

Reply:
No, it's needed in other contexts too, e.g. choosing which key to update, whether to create/update/delete.  

(Also, it doesn't really fit with the name/purpose of the type `MapBias` -- it's a better fit as part of `hammerState`)